src/stores/notifications/SummarizedNotificationState.ts

```diff
@@ -27,23 +26,22 @@ import { RoomNotificationState } from "./RoomNotificationState";
  */
 export class SummarizedNotificationState extends NotificationState {
     private totalStatesWithUnread = 0;
-    unreadRooms: Room[];
+    private unreadRoomId: string;
```

Comment:
A `SummarizedNotificationState` should know nothing about rooms or room notification states. The docs, class, etc are all intended for generalized use and not necessarily rooms. If we need to, we'd create a `SummarizedRoomNotificationState`, though this feels overkill. Is there a better place we can put this stuff than the notification state classes?

Reply:
Oh, I see. I'm not sure if there's a better place to put it. As `notificationState` contains the information about the count of `unreadRooms` and listens to updates, I thought that was the best place. If there's any better way, I'll use that. Otherwise, I'll go with `SummarizedRoomNotificationState`, but I'm not sure if that's desired.

Reply:
@t3chguy do you know of any other ways to get the room id for the home space? If not I'll go with the above method.

Reply:
the Home space isn't an actual space and as such it doesn't have a room ID; in the Space Store it is represented by null or the HOME_SPACE symbol depending on the context.

Reply:
Oops, sorry I phrased it wrong. I was asking about the first unread room id inside the home space?

Reply:
Probably something like https://github.com/matrix-org/matrix-react-sdk/blob/a839d0f3963eb00f22db2f205c4d285d34125a3a/src/components/views/rooms/RoomSublist.tsx#L427-L450 but having to consider `TAG_ORDER` and `feature_spaces.all_rooms`

Reply:
Okay, will do that